### DISCUSSION TITLE
Error while Downloading Prisma Engines for Node-API on Windows

### DISCUSSION THREAD

**Discussion by OshriAsulin** [prisma/prisma#19915] [Q&A]
### Bug description

npx prisma init
> Downloading Prisma engines for Node-API for windows [                    ] 0%Error: request to https://binaries.prisma.sh/all_commits/b20ead4d3ab9e78ac112966e242ded703f4a052c/windows/query_engine.dll.node.gz.sha256 failed, reason: unable to get local issuer certificate

### How to reproduce

I try to run - "npx prisma init"

 Downloading Prisma engines for Node-API for windows [                    ] 0%Error: request to https://binaries.prisma.sh/all_commits/b20ead4d3ab9e78ac112966e242ded703f4a052c/windows/query_engine.dll.node.gz.sha256 failed, reason: unable to get local issuer certificate

### Expected behavior

Prisma will add the folder of the files to be mapped to the database

### Prisma information

```prisma
// Add your schema.prisma
```

```ts
// Add your code using Prisma Client
```

### Environment & setup

- OS: 
- Database: 
- Node.js version: 

### Prisma Version

```

```

**Comment by janpio** [comment]
Can you download the file from the URL in your browser? Or with `curl`?

**Reply by OshriAsulin** [reply]
Yes I downloaded it, but what should be done with it?

**Reply by janpio** [reply]
Nothing, this should not be necessary.

You can use https://www.prisma.io/docs/reference/api-reference/environment-variables-reference#prisma_engines_checksum_ignore_missing to skip this specific download, but then the actual download will probably fail with the same error message. 

If that still happens, you can then download that file manually and use https://www.prisma.io/docs/reference/api-reference/environment-variables-reference#prisma_query_engine_library to define the local location of where you put the file.

**Reply by nurul3101** [reply]
Marking this discussion as resolved as this discussion seems to be a duplicate of support ticket: [#2817](https://prisma1015.zendesk.com/agent/tickets/2817)
The issue is resolved as confirmed by the author.

**Reply by Trixtoparadise** [reply]
> Nothing, this should not be necessary.
> 
> You can use https://www.prisma.io/docs/reference/api-reference/environment-variables-reference#prisma_engines_checksum_ignore_missing to skip this specific download, but then the actual download will probably fail with the same error message.
> 
> If that still happens, you can then download that file manually and use https://www.prisma.io/docs/reference/api-reference/environment-variables-reference#prisma_query_engine_library to define the local location of where you put the file.

How do you go about doing this?

**Comment by aryan0-hash** [comment]
@nurul3101 @janpio @OshriAsulin i'm also facing the same issue can someone help please i got struck in this since two week I have all the thing which is present in internet but still not working please help me?

**Reply by janpio** [reply]
Open a new discussion and actually describe your problem with as much information as possible. Thanks.

**Comment by DoudGaya** [comment]
I had the same issue. This is how I fix it.

This error has nothing to do with Prisma or Node. It's from your firewall rules on Windows.

If you're on Windows 11 or 10, Go to `Settings > Network and Internet > Advanced Network Settings > Network Reset`. Wait for your PC to restart, and then when you receive a prompt about a firewall, accept it.

**Reply by Sobaken9D** [reply]
After 2 days it worked... thank you
